diff --git a/node_modules/prismjs/components.json b/node_modules/prismjs/components.json
index ac6ad14..8401713 100644
--- a/node_modules/prismjs/components.json
+++ b/node_modules/prismjs/components.json
@@ -518,6 +518,10 @@
 			"require": "c",
 			"owner": "RunDevelopment"
 		},
+    "hoon": {
+      "title": "Hoon",
+      "owner": "matildepark"
+    },
 		"http": {
 			"title": "HTTP",
 			"optional": [
diff --git a/node_modules/prismjs/components/prism-hoon.js b/node_modules/prismjs/components/prism-hoon.js
new file mode 100644
index 0000000..b5f5a79
--- /dev/null
+++ b/node_modules/prismjs/components/prism-hoon.js
@@ -0,0 +1,30 @@
+Prism.languages.hoon = {
+  constant: /(?:%\.n)|(?:%\.y)|(?:%[\w-]+\w?)/,
+  comment: {
+    pattern: /::.*/,
+    lookahead: true,
+    greedy: true
+  },
+  "function-name" :/(?:\+[-+]  )?(?:[a-z]([a-z0-9-]*[a-z0-9])?)/,
+  "class-name": [
+    {
+    pattern: /@(?:[a-z])?(?:[A-Za-z0-9-]*[A-Za-z0-9])?/,
+    lookahead: true,
+    greedy: true
+    },
+    /\*/
+  ],
+  "string": [
+    {
+      pattern: /"(?:[^"])*"/,
+      lookahead: true,
+      greedy: true
+    },
+    {
+      pattern: /'(?:[^'])*'/,
+      lookahead: true,
+      greedy: true
+    }
+  ],
+  "keyword": /:_|\.[\^\+\*=\?]|![><:\.=\?!]|=[>|:,\.\-\^<+;/~\*\?]|\?[>|:\.\-\^<\+&~=@!]|\|[\$_%:\.\-\^~\*=@\?]|\+[|\$\+\*]|:[_\-\^\+~\*]|%[_:\.\-\^\+~\*=]|\^[|:\.\-\+&~\*=\?]|\$[|_%:<>\-\^&~@=\?]|;[:<\+;\/~\*=]|~[>|\$_%<\+\/&=\?!]|--|==/
+}
\ No newline at end of file
